packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-listener.ts

```diff
@@ -813,6 +814,22 @@ export interface AddApplicationTargetsProps extends AddRuleProps {
    */
   readonly stickinessCookieDuration?: Duration;
 
+  /**
+   * The app cookie name.
+   *
+   * Indicates the name of the application-based stickiness cookie.
+   *
+   * Names that start with the following prefixes are not allowed: AWSALB, AWSALBAPP,
+   * and AWSALBTG; they're reserved for use by the load balancer.
+   *
+   * Note: `stickinessCookieName` parameter depends on the presence of `stickinessCookieDuration` parameter.
+   * If `stickinessCookieDuration` is not set, `stickinessCookieName` will be omitted.
+   *
+   * @default No app cookie name
```

Comment:
How about something like this?

```suggestion
   * @default - If `stickinessCookieDuration` is set, a load-balancer generated cookie is used. Otherwise, no stickiness is defined.
```